opentelemetry-semantic-conventions/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## vNext
 
+## v0.26.0
+### Changed
+
+- Starting with this version, this crate will use Weaver for the generation of
```

Comment:
Could you also include a note on `semconv_experimental`?

Reply:
Added a short description in the changelog. See [c6af804](https://github.com/open-telemetry/opentelemetry-rust/pull/2098/commits/c6af8048553a39eee58f93a737437d80f9537a0e)